src/ViewInstance.js

```diff
@@ -103,6 +109,19 @@ class ViewInstance extends React.Component {
     this.calloutRef = createRef();
   }
 
+  componentDidMount() {
+    this.getMARCRecord();
+  }
+
+  getMARCRecord = () => {
```

Comment:
Please make it a regular method (not an arrow function) as it does not need to be bound.